.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: ci
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  tests:
+    name: tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.20"
+          
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Force Failure
```

Comment:
The 'Force Failure' step seems to be a placeholder. It would be better to replace it with actual tests or remove it if it's not needed. [important]